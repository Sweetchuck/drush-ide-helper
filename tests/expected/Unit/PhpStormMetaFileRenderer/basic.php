<?php

/**
 * @file
 * PhpStorm meta.
 */

namespace PHPSTORM_META {

  override(
    \Ab\Cd::e(0),
    map([
      'f' => \Ab\Cd\FInterface::class,
    ])
  );

  override(
    \Gh\Ij::k(1),
    map([
      'l' => \Gh\Ij\LInterface::class,
      'm' => \Gh\Ij\MInterface::class,
      'n' => \Gh\Ij\NInterface::class,
    ])
  );

}
