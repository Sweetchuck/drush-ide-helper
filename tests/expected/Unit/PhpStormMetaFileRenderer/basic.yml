-
  - '\Ab\Cd'
  - 'e(0)'
  -
    f: '\Ab\Cd\FInterface'

-
  - '\Gh\Ij'
  - 'k(1)'
  -
    l: '\Gh\Ij\LInterface'
    m: '\Gh\Ij\MInterface'

-
  - '\Gh\Ij'
  - 'k(1)'
  -
    n: '\Gh\Ij\NInterface'
